platform/ios/src/MGLMapView.mm

```diff
@@ -4156,7 +4151,12 @@ - (MGLAnnotationTag)annotationTagAtPoint:(CGPoint)point persistingResults:(BOOL)
             {
                 if ([annotation isKindOfClass:[MGLMultiPoint class]])
                 {
-                    return false;
+                    if ([self.delegate respondsToSelector:@selector(mapView:shapeAnnotationIsEnabled:)] &&
+                        ![self.delegate mapView:self shapeAnnotationIsEnabled:(MGLMultiPoint *)annotation]) {
+                        return true;
+                    } else {
+                        return false;
```

Comment:
This is fine, but FYI you can also wrap the conditional in `!!` or compare to `YES`. (×2)